database/src/main/postgres/runs/additional_data.ddl

```diff
@@ -13,12 +13,10 @@
  * limitations under the License.
  */
 
--- DROP TABLE IF EXISTS runs.additional_data;
-
 CREATE TABLE runs.additional_data
 (
     id_additional_data      BIGINT NOT NULL DEFAULT global_id(),
-    key_segmentation        BIGINT NOT NULL,
+    fk_partitioning         BIGINT NOT NULL,
     ad_name                 TEXT NOT NULL,
     ad_value                TEXT,
```

Comment:
I thought that this will be a JSONB, why not?

Reply:
I'm not sure if I understood you to be honest, apologies. I was referring to `ad_name` and `ad_value` - so this means that a sinle record in `additional_data` table will hold only 1 metadata record that is 'flat' like this. I thought that a single additional_data record should be a JSON structure-check our data model:
```
case class AdditionalDataDTO(
  additionalData: Map[String, Option[String]]
)
```

Reply:
It's decomposed for better handling. Indexing, searching, patterning and merging.
If the DB is to understand the data (do operations on them), I really prefer if they are not hidden in complex types.

Reply:
Okay then. Let's use 'flat' data types whenever possible. In case of partitioning and measurements - those could also be decomposed, but let's leave it all as is, since we'll need some flexibility given the current state of the project (and already existing implementation)